addon/TabTracker.js

```diff
@@ -43,6 +35,14 @@ TabTracker.prototype = {
     this._trackableURLs = trackableURLs;
     this._experimentID = experimentId;
     this._store = store;
+
+    this.onOpen = this.onOpen.bind(this);
```

Comment:
Technically this could stay in the constructor, although I guess it's fine to have it here